### PR TITLE
Repeats: Support repeating HTML

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: server/chat-plugins/repeats.ts [hunk 1]
```diff
@@ -155,19 +157,20 @@ export const commands: ChatCommands = {
 
 		Repeats.addRepeat(room, {
```
Comment (AnnikaCodes) [inline server/chat-plugins/repeats.ts:158]:
```suggestion

		this.checkHTML(message);
		Repeats.addRepeat(room, {
```

File: server/chat-plugins/repeats.ts [hunk 2]
```diff
@@ -155,19 +157,20 @@ export const commands: ChatCommands = {
...
 		this.privateModAction(
-			room.tr`${user.name} set the phrase "${message}" to be repeated every ${interval} minute(s).`
+			room.tr`${user.name} set the phrase labeled with "${id}" to be repeated every ${interval} minute(s).`
 		);
 	},
 	repeathelp() {
 		this.runBroadcast();
 		this.sendReplyBox(
 			`<code>/repeat [minutes], [id], [phrase]</code>: repeats a given phrase every [minutes] minutes.<br />` +
+			`<code>/repeathtml [minutes], [id], [phrase]</code>: repeats a given phrase every [minutes] minutes. Requires: # &<br />` +
```
Comment (AnnikaCodes) [inline server/chat-plugins/repeats.ts:173]:
```suggestion
			`<code>/repeathtml [minutes], [id], [phrase]</code>: repeats a phrase containing HTML every [minutes] minutes. Requires: # &<br />` +
```

File: server/chat-plugins/repeats.ts [hunk 3]
```diff
@@ -138,9 +138,11 @@ export const pages: PageTable = {
 };
 
 export const commands: ChatCommands = {
-	repeat(target, room, user) {
+	htmlrepeat: 'repeat',
```
Comment (AnnikaCodes) [inline server/chat-plugins/repeats.ts:141]:
```suggestion
	repeathtml: 'repeat',
```

**Review by AnnikaCodes — APPROVED** [review]
Looks good once the merge conflicts are resolved! Thanks!

**Comment by Zarel** [comment]
tyty

**Pull request merged**